cmd/thv/app/list.go

```diff
@@ -66,7 +66,7 @@ func listCmdFunc(cmd *cobra.Command, _ []string) error {
 	}
 
 	if len(toolHiveContainers) == 0 {
-		logger.Infof("No MCP servers found")
+		logger.Info("No MCP servers found")
```

Comment:
Not directly related to the original issue, but use `Info` here since the logger expects extra parameters for `Infof`, which leads to some junk in the output.

Reply:
We probably want to replace that for a simple fmt.Printf since it's no use printing the log level.